xmessage.yaml

```diff
@@ -0,0 +1,57 @@
+# Generated from https://git.alpinelinux.org/aports/plain/community/xmessage/APKBUILD
```

Comment:
we can remove this line

Reply:
i think its ok to keep this as we have this in other packages.